Feature/readme

cagallo opened the pull request:
**What (if any) features are you implementing?**
 -Add README(still needs GIF but want to have it fully refactored before we do that)
-Add title to HTML
 
**What (if anything) did you refactor?**
 -None
 
**Were there any issues that arose?**
-None

**Is there anything needs to be prioritized?**
 -none
 
**Is there anything that you need from your teammates?**
 -nothing!
 
**GOALS?**
 -Finishing this project!!

cagallo committed:
Add working README

cagallo committed:
Add title to html

ccollums commented on README.md:
```diff
@@ -0,0 +1,25 @@
+- Filter through your saved ideas.
+- View your saved ideas all at once.
+
+##GitHub Pages
```
Need to add all of our github links to our separate pages

ccollums commented on README.md:
```diff
@@ -0,0 +1,25 @@
+
+##GitHub Pages
+
+<a href="urlhere">Click me</a> to go to the site.
```
Is this for the deployment link or just the repo link? Probably need both!

ccollums commented on README.md:
```diff
@@ -0,0 +1,25 @@
+#Ideabox Group Project
+Create, display, record and archive idea cards.
+
```
need to add installation instructions here

ccollums commented on README.md:
```diff
@@ -0,0 +1,25 @@
+#Ideabox Group Project
+Create, display, record and archive idea cards.
+
+##Description
```
add technologies/languages used

ccollums reviewed (commented):
Looks good Chez! Love the description. We need to add a couple more things but looking good!

cagallo committed:
Update README.md

-added installation instructions and link to repo(lines 4-11)
-added link to actual site (line 28)
-added links to author's GitHub accounts (lines 32-34)

cagallo committed:
Update README.md

-Add languages used

ccollums reviewed (commented):
Well done!

michelecomfort reviewed (commented):
Looks great!
